### ISSUE TITLE
Chemical names in PDOM should not be capitalized

### ISSUE THREAD

**Issue by emily-phet** [phetsims/molarity#88]
For example - "potassium permanganate", "water", etc. Currently they have sentence case (probably being pulled directly from the labels in the combobox) but in the prom they should be all lower case.
